src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Test/ScenarioTests/IaasVm/ContainerTests.ps1

```diff
@@ -14,6 +14,17 @@
 
 function Test-GetContainerScenario
 {
-	$vaults = Get-AzureRmRecoveryServicesVault -ResourceGroupName "phaniktRSV" -Name "phaniktRs1";
-	echo $vaults;
+	$vault = Get-AzureRmRecoveryServicesVault -ResourceGroupName "phaniktRSV" -Name "phaniktRs1";
+	$containers = Get-AzureRmRecoveryServicesContainer -Vault $vault -ContainerType "AzureVM" -Status "Registered";
```

Comment:
can you define variables for rgname, name? also trace getVault successful for,  get containers successful for etc